### DISCUSSION TITLE
License?

### DISCUSSION THREAD

**Discussion by Techokami** [TrianglyRU/OrbinautFramework#22] [Q&A] [answered]
I would have made this an issue on your issue tracker, but you've disabled that, so I'll have to do it here then.
Where is the license for this project? Not having one can make things rather hairy for people wanting to use the framework for something beyond just Sonic fangames. I'm going by experience here; Sonic Worlds Delta originally didn't have a proper license, and it caused weird situations for Freedom Planet and Spark the Electric Jester. (These were resolved by relicensing to BSD 3-Clause)

Personally I would suggest using either the MIT License, or one of the BSD licenses, as they are adjacent to the fangame community's guidelines of giving credit for using things from others, but would also keep things clear for non-fangame use cases.

**Comment by TrianglyRU** [comment, marked as answer]
To be honest, I don't really care about the licenses for Orbinaut (which can be kinda unprofessional by me). They sound way too legal so it is simply how it was for OF1, which I forgot to include in the ReadMe: for Sonic games, it obviously should not be profitable, and for original IPs, you must follow GameMaker's Licenses. 

But thank you so much for the suggestion! I'll probably use the MIT License.
